public/editor/stylesheets/editor.less

```diff
@@ -652,6 +652,13 @@ body {
       }
     }
 
+    &:hover {
+      color: rgba(0,0,0,.6);
+      .checkbox {
```

Comment:
Can you remove the border rule from the hover effect please? The checkbox already has that styling so it's not necessary.